File: effective/src/com/lab/java/zayson/item65/Car.java
package com.lab.java.zayson.item65;

public class Car {
    private final String name;
    private int speed;
    public Car(String name, int speed) {
        this.name = name;
        this.speed = speed;
    }

    public String getName() {
        return name;
    }

    public void speedUp() {
        speed++;
    }

    public int getSpeed() {
        return speed;
    }
}

File: practice/src/main/java/zayson/java/lab/kotlin/grammer/lecture11/Car.kt
package zayson.java.lab.kotlin.grammer.lecture11

class Car private constructor(
    internal val name: String,  // name의 getter를 internal로 선언
    private var owner: String,  // owner의 getter/setter를 private으로 선언
    _price: Int
) {
    var price = _price
        private set  // getter의 경우 public, setter의 경우 private으로 지정

}